service-worker.js

```diff
@@ -0,0 +1,29 @@
+// START OF AUTO-GENERATED SW-TOOLBOX BY SW-PRECACHE CLI
+
+/*
+ Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Is this expected to be 2016? Does it update to 2017 if you re-generate it again?